### DISCUSSION TITLE
ERROR - Integrity constraint violation

### DISCUSSION THREAD

**Discussion by ggalvin-sc** [kimai/kimai#4515] [On-Premise: Installation & Updates]
When I try to install with the bin/console kimai:install -n command I get the following error on my Ubuntu Linux install.  This should not be a user permission problem as I set the Mysql User to a superuser for troubleshooting.  Any help would be great.  Thanks again. 

Kimai installation running ...
==============================

Database `database` for connection named default already exists. Skipped.
                                                                                                                        
 [ERROR] Failed to set migration status: An exception occurred while executing a query: SQLSTATE[23000]: Integrity      
         constraint violation: 1052 Column 'TABLE_NAME' in where clause is ambiguous

**Comment by eduardogiorello** [comment]
Mismo error. Encontraste solución

**Comment by ggalvin-sc** [comment]
No

Gregory Michael Galvin

**Comment by rodrigueznicolas** [comment]
@ggalvin-sc @eduardogiorello I've add the same problem, and it was coming from the server version in the env DATABASE_URL variable.

**Reply by kevinpapst** [reply]
Good finding, will add it to the Installation "Troubleshooting" section, thanks for sharing @rodrigueznicolas 👍

**Reply by eduardogiorello** [reply]
muchas gracias
